pandas/core/strings.py

```diff
@@ -2145,11 +2145,61 @@ def rindex(self, sub, start=0, end=None):
 
     _shared_docs['casemethods'] = ("""
     Convert strings in the Series/Index to %(type)s.
+
     Equivalent to :meth:`str.%(method)s`.
 
     Returns
     -------
-    converted : Series/Index of objects
+    Series/Index of objects
+
+    See Also
+    --------
+    Series.str.lower : Converts all characters to lower case.
```

Comment:
"lower case" -> "lowercase" ? 
Or at least we should be consistent with what is substituted in the summary line (currently this is "lowercase")